docs/source/paper_index.md

```diff
@@ -5,8 +5,28 @@
 
 ## Group Relative Policy Optimization
 
-Papers relating to the [`GRPOTrainer`]
+### DeepSeekMath: Pushing the Limits of Mathematical Reasoning in Open Language Models
+**📜 Paper**: https://huggingface.co/papers/2402.03300
 
+Introduces **GRPO** and shows strong math-reasoning gains from math-centric pretraining plus group-relative PPO-style optimization.  
+**Used in TRL via:** [`GRPOTrainer`]
+
+```python
+# Minimal GRPO setup (mirrors style used for other papers on the page).
+from trl import GRPOConfig
+
+training_args = GRPOConfig(
+    loss_type="grpo",
+    beta=0.0,                 # GRPO commonly trains without explicit KL in released configs
+    epsilon=2e-4,             # clip range (use paper/experiment settings if you mirror them)
+    epsilon_high=4e-4,        # upper clip (symmetrical if not specified)
```

Comment:
The goal here is to roughly match the setting used in the paper. So similarly to beta: in the original paper, they don't use asymmetric clipping